matlab/lib/+Ice/+SSL/ConnectionInfo.m

```diff
@@ -18,7 +18,7 @@
                 connectionId = '';
                 peerCertificate = '';
             end
-            obj = obj@Ice.ConnectionInfo(underlying, incoming, adapterName, connectionId);
+            obj@Ice.ConnectionInfo(underlying, incoming, adapterName, connectionId);
```

Comment:
The `obj = obj@...` syntax works fine but looks strange with multiple inheritance, since you assign obj twice (or more).